validation/static/menu.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+function showMenu() {
+    let menu = document.getElementById("options-menu");
+    if (menu.classList.contains("menu__none")) {
+        menu.classList.remove("menu__none");
+        menu.classList.add("menu__block");
+    } else {
+        menu.classList.add("menu__none");
+        menu.classList.remove("menu__block");
+    }
+}
+
+function showMacron(option) {
+    document.cookie = "macron=" + option + "; SameSite=Lax;"
```

Comment:
Why `SameSite=Lax`?

Reply:
If I don't provide any sort of CORS information, I get a warning saying that the cookie will expire soon and this gets rid of the warning. I'll do some research to see if there's a better option, though.

Reply:
I think this is actually what we want: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite#lax

Lax is the new default and makes sure the cookie is passed when following a link to the page.